src/Parser/Rogue/Outlaw/Modules/Features/AlwaysBeCasting.js

```diff
@@ -0,0 +1,60 @@
+import CoreAlwaysBeCasting from 'Parser/Core/Modules/AlwaysBeCasting';
+import SPELLS from 'common/SPELLS';
+
+class AlwaysBeCasting extends CoreAlwaysBeCasting {
+
+    static BASE_GCD = 1000;
+    static MINIMUM_GCD = 1000;
+    
+    static ABILITIES_ON_GCD = [
+        SPELLS.AMBUSH.id,
+        SPELLS.BLUNDERBUSS.id,
+        SPELLS.PISTOL_SHOT.id,
+        SPELLS.SABER_SLASH.id,
+
+        SPELLS.DEATH_FROM_ABOVE_TALENT.id,
+
+        SPELLS.RUN_THROUGH.id,
+        SPELLS.BETWEEN_THE_EYES.id,
+        SPELLS.ROLL_THE_BONES.id,
+        SPELLS.SLICE_AND_DICE_TALENT.id,
+
+        SPELLS.CRIMSON_VAIL.id,
```

Comment:
should be `CRIMSON_VIAL`, but I guess this spell was already in our listing with that misspelling... I guess the typo is following precedent :smile: